imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -2519,7 +2519,7 @@ func Test_fdSeek(t *testing.T) {
 				'?',
 			},
 			expectedLog: `
-==> wasi_snapshot_preview1.fd_seek(fd=4,offset=4,whence=0,4557430888798830399)
```

Comment:
just for anyone curious: basically this weird number is the u64 value on mem[newoffset]  _before_ we execute fd_seek.

Reply:
which is useless and meaningless in any way